src/NuGet.Core/NuGet.Common/Errors/NuGetLogCode.cs

```diff
@@ -825,6 +825,11 @@ public enum NuGetLogCode
         /// </summary>
         NU5131 = 5131,
 
+        /// <summary>
+        /// Projects that use central packge version management should not define the verison on the PackageReference items but on the PackageVersion items.
+        /// </summary>
```

Comment:
@anangaur please review the error code.

Reply:
The error codes are engineering managed. 

The top of this class has code guidelines https://github.com/NuGet/NuGet.Client/blob/715edd1291b089d42cfef65566146f45a8cd6310/src/NuGet.Core/NuGet.Common/Errors/NuGetLogCode.cs#L22-L34

You are probably looking at NU1007.

Reply:
Changed to NU1008.